src/mdoc/model/DeviceResponse.ts

```diff
@@ -94,18 +77,22 @@ export class DeviceResponse {
    * @param {Buffer} sessionTranscriptBytes - The sessionTranscriptBytes data to use in the session transcript.
    * @returns {DeviceResponse}
    */
-  public usingSessionTranscriptBytes(sessionTranscriptBytes: Buffer): DeviceResponse {
+  public usingSessionTranscriptBytes(sessionTranscriptBytes: Buffer | Promise<Buffer>): DeviceResponse {
     if (this.sessionTranscriptBytes) {
       throw new Error(
         'A session transcript has already been set, either with .usingSessionTranscriptForOID4VP, .usingSessionTranscriptForWebAPI or .usingSessionTranscriptBytes',
       );
     }
-    this.sessionTranscriptBytes = sessionTranscriptBytes;
+    if (sessionTranscriptBytes instanceof Buffer) {
+      this.sessionTranscriptBytes = Promise.resolve(sessionTranscriptBytes);
+    } else {
+      this.sessionTranscriptBytes = sessionTranscriptBytes;
+    }
```

Comment:
is this change necessary?

Reply:
Indeed, the field is awaited everywhere. I removed this check in 40bc700